quadra/tasks/patch.py

```diff
@@ -341,6 +330,18 @@ def test(self) -> None:
             self.idx_to_class[i] if i != -1 else "N/A" for i in res["real_label"].unique().tolist()
         ]
 
+    @property
+    def deployment_model(self):
+        """Deployment model."""
+        return None
+
+    @deployment_model.setter
+    def deployment_model(self, model_path: str):
```

Comment:
Same here, I would probably expect that we can deploy the model without a backbone, but if a deployed model is present it should be the first option